resources/model/OpenStudio.idd

```diff
@@ -13766,19 +13766,22 @@ OS:Boiler:HotWater,
   N8, \field Parasitic Electric Load
        \type real
        \units W
-        \ip-units W
+       \ip-units W
        \minimum 0.0
        \default 0.0
   N9, \field Sizing Factor
        \note Multiplies the autosized capacity and flow rates
        \type real
        \minimum> 0.0
        \default 1.0
-  A9 ; \field End-Use Subcategory
+  A9, \field End-Use Subcategory
        \note Any text may be used here to categorize the end-uses in the ABUPS End Uses by Subcategory table.
        \type alpha
        \retaincase
        \default General
+  N10; \field Off Cycle Parasitic Fuel Load
+       \units W
+       \note parasitic fuel load when the boiler is not operating (i.e., standing pilot)
```

Comment:
Add required-field. And adjust \min-fields above.

----

Following E+ and putting it as the end. I know this is less work because no VT needed, but it's lazy. It should have been next to the On Cycle Parasitic Electric Load in E+.

Perhaps we should do it anyways.

Reply:
That was me. Yes, I was being ~lazy~ efficient.

Reply:
Can't blame you. Especially since Transition in E+ forces you to go to fortran hell :)
For us, VT will need to happen anyways to hard code it to zero, so it's two lines extra to move it in the right location